package/CHANGELOG

```diff
@@ -21,6 +21,7 @@ The rules for this file:
   * 2.0.0
 
 Fixes
+  * Fixed UnboundLocalError in fasta2align function (Issue #3124)
```

Comment:
Can you add yourself to the line above (on line 19).

Reply:
Fixed issue with unassigned 'GAP' variable when fasta2algin function gets a no gap residue input. Does this seem okay?